{
  "bug_id": "api-get-api-reports-summary-1761191876705",
  "workflow_id": "api-get-api-reports-summary",
  "category": "api_endpoint_error",
  "priority": 65,
  "evidence": {
    "workflow_id": "api-get-api-reports-summary",
    "started_at": "2025-10-23T03:57:56.703Z",
    "steps": [],
    "requests": [],
    "responses": [],
    "failed_at": "2025-10-23T03:57:56.705Z",
    "duration_ms": 2,
    "error": "request to http://localhost:3002/api/reports/summary failed, reason: "
  },
  "discovered_at": "2025-10-23T05:46:27.423Z"
}